### PR TITLE
Added 5 more Next.js courses to free-courses-hi.md

### PR DESCRIPTION
**Pull request by Akshat3144**
## What does this PR do?
Add resource(s)

### Description
Added 5 more Next.js courses to the `free-courses-hi.md` file.

Watch links (more than 2 hours each) ->
- Learn NextJS with Aceternity UI library by building a project : Chai aur Code
- Master Next JS 14 Complete Basic to Advance : Geeky Shows

Playlist link ->
- NextJS Tutorial with Project in Hindi : Learn Code With Durgesh
- NextJS Master Course : Piyush Garg
- Next.JS Full FREE Course (New Updated) by WsCube Tech : WsCube Tech

### Why is this valuable (or not)?
These additional resources provide more options for learning Next.js in Hindi, catering to a wider audience.

### How do we know it's really free?
All the added courses are available for free on YouTube.

### For book lists, is it a book? For course lists, is it a course? etc.
These are video courses.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### Index
 
 * [Algorithms](#algorithms)
-    * [Soft Computing](#soft-computing)
+  * [Soft Computing](#soft-computing)
```
Comment (eshellman) [inline courses/free-courses-hi.md:4]: remove the format changes

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -82,14 +83,14 @@
 * [Binary Trees - by LoveBabbar](https://www.youtube.com/playlist?list=PLDzeHZWIZsTo87y1ytEAqp7wYlEP3nner) - CodeHelp - by Babbar
 * [Binary Trees - Level 1](https://www.youtube.com/playlist?list=PL-Jc9J83PIiHYxUk8dSu2_G7MR1PaGXN4) - Pepcoding
 * [Bit Manipulation - Level 2](https://www.youtube.com/playlist?list=PL-Jc9J83PIiFJRioti3ZV7QabwoJK6eKe) - Pepcoding
-* [Complete C++ DSA Course \| Data Structures & Algorithms Playlist](https://www.youtube.com/playlist?list=PLfqMhTWNBTe137I_EPQd34TsgV6IO55pt) - Apna College
+* [Complete C++ DSA Course \| Data Structures &amp; Algorithms Playlist](https://www.youtube.com/playlist?list=PLfqMhTWNBTe137I_EPQd34TsgV6IO55pt) - Apna College
```
Comment (eshellman) [inline courses/free-courses-hi.md:86]: markdown, not html

**Review by eshellman — CHANGES_REQUESTED** [review]
can't see the content changes through the forest of formatting changes.

**Comment by Akshat3144** [comment]
Thanks sir, All requested changes Done, please check.

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -74,6 +74,7 @@
 * [TypeScript](#typescript)
 * [WordPress](#wordpress)
 
+---
```
Comment (eshellman) [inline courses/free-courses-hi.md:77]: please remove these

**Comment by Akshat3144** [comment]
requested change done, sir, please check.

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -512,11 +512,16 @@
 
 #### Next.js
 
+* [Learn NextJS with Aceternity UI library](https://www.youtube.com/watch?v=cVKB5NQPiFA&t=744s) - Chai aur Code
```
Comment (eshellman) [inline courses/free-courses-hi.md:515]: Don't invent titles! Use the title given by the resource.

**Review by eshellman — CHANGES_REQUESTED** [review]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
"watch" links are ok if they are more than an hour and have outlines. To make it easier for the reviewers, please note this in your PR description.

**Comment by Akshat3144** [comment]
Added the official titles and updated the description of the pull request.

**Comment by Akshat3144** [comment]
Hey, could you add hacktoberfest label over it because apparently, its not updated there. Thanks

**Comment by eshellman** [comment]
the whole repo has the label

**Pull request merged**